config/constants.ts

```diff
@@ -98,6 +97,9 @@ export const getEnvironmentConfig = (
             securityGroupProps: {
               ...orcaBusStatefulConfig.securityGroupProps,
             },
+            eventSourceProps: {
```

Comment:
IIRC the event sources will eventually have to be dynamic? As in not requiring a de-deploy when we want to monitor and ingest new buckets... I guess that's a future feature, but just keeping tabs open on the topic as I guess that this will eventually have to go to some Rust AWS SDK lambda instead of CDK? ... thinking out loud, no need to address this in this PR.

Reply:
Hmm, could potentially have env variables that contain an SQS url that the Lambda function uses. Although, wouldn't every new bucket need to somehow be registered with the SQS queue, requiring CDK? Not sure how SQS could discover these on its own, unless there's just a blanket "all buckets" EventBridge rule.

Reply:
Exactly, @reisingerf, what were your thoughts from previous meetings on this? I'm not sure what are the requirements for this.

Reply:
The idea is that buckets of interest are configured to send events to the EventBus, to make those discoverable. 

Marko's statefull stack will contain (and be expanded with) Event Rules that then funnel events from those buckets of interest to the SQS queue. 
The stateless FM stack does not need to change at all for this, as it simply consumes events from the SQS. 

Does that make sense?

Reply:
Ok, so the dev story of adding a new "bucket of interest" is declaring it statically in CDK (on the EventBus construct/stack in this case) or those Event Rules will be lax enough to just process events from a new bucket? Or some other mechanism I'm not understanding? :-S

Reply:
"adding a new bucket" is simply adding a new Rule to the stateless stack to forward the events to SQS. This should always be possible, even if there is no such bucket (as it's a simple pattern matching on the EventBus side). 

How actual events get to the EventBus is a different story and outside the scope of the stack. This would be the responsibility of whoever controls the bucket in question.

Reply:
Could be nice to have the ability to add multiple EventBridge rules to the same SQS queue within this PR. Not sure if this is super useful though because multiple buckets can already be defined for the same rule.

Reply:
Hm, about that... 
I need to brush up on the Rule syntax, but if I understand the code correctly then the rule filters for a known list of buckets and a given prefix, right? 

That prefix would have to be the same across all the listed buckets though? 
If yes, then I think I'd prefer a rule per bucket / concern... easier to keep track of and maintain independence.

Reply:
Yes, it's one prefix for all the buckets. I think you're right, it might make more sense to have it defined per bucket.